trino/trino_test.go

```diff
@@ -1719,3 +1719,52 @@ func BenchmarkQuery(b *testing.B) {
 		rows.Close()
 	}
 }
+
+func TestExec(t *testing.T) {
```

Comment:
I would argue this does not test INSERT (what it does per commit title) - give no support of that in connector.

Reply:
Sorry, I don't understand. The memory connector supports inserts but doesn't support updates and deletes. And this tests executes an `INSERT` query, and passes.